api_app/services/azure_resource_status.py

```diff
@@ -27,5 +27,9 @@ def get_azure_resource_status(resource_id):
 
 
 def get_azure_vm_instance_view(vm_name, resource_group_name) -> models.VirtualMachineInstanceView:
-    compute_client = ComputeManagementClient(credentials.get_credential(), config.SUBSCRIPTION_ID)
+    resource_manager_endpoint = cloud.get_cloud().endpoints.resource_manager
+    compute_client = ComputeManagementClient(credentials.get_credential(),
+                                             subscription_id=config.SUBSCRIPTION_ID,
+                                             base_url=resource_manager_endpoint,
+                                             credential_scopes=[resource_manager_endpoint + ".default"])
```

Comment:
Looks like we should create a method to return the credential scopes instead of doing this 3 times.